ads/sekindo.md

```diff
@@ -0,0 +1,33 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Sekindo
+
+Please visit our web (http://www.sekindo.com) for more details
+## Example
+
+```html
+<amp-ad width="300" height="250"
+    type="sekindo"
+    data-spaceId="14071">
+</amp-ad>
+```
+
+## Configuration
+
+__Required:__
+
+- `data-spaceId` - Adunit unique id
```

Comment:
Just FYI: 
`data-spaceId` will be converted to `data['spaceid']`
`data-space-id` will be converted to `data['spaceId']`